Refine grammar and style in the documentation's language section. (#39)
tact-lang/tact-docs: pages/language/guides/functions.mdx

## Mutable functions

> Mutable functions are performing mutation of a value replacing it with an execution result. To perform the mutation function must change the `self` value.

Reviewer: ```suggestion
Mutable functions are performing mutation of a value replacing it with an execution result. To perform mutation, the function must change the `self` value.
```

## Native functions

> Native functions are directly binding to a func function:

Reviewer: ```suggestion
Native functions are direct bindings of func functions:
```

> **Note**

> > Native functions could be also mutable and extensions of one

Reviewer: ```suggestion
> Native functions could be also mutable and extension ones.
```

## Getter Functions

> Getter functions that define getters on smart contracts and can be defined only within a contract or trait.

Reviewer: ```suggestion
Getter functions define getters on smart contracts and can be defined only within a contract or trait.
```
